app/shop/tests.py

```diff
@@ -1,100 +0,0 @@
-from datetime import datetime
-from decimal import Decimal
-from django.db.models import Q
-from django.test import TestCase
-from django.utils import timezone
-
-from .functions import SumIf, TruncQtr, TruncQuarter, qtr_over_qtr_revenue
-from .models import Product, Sale
-
-
-class ShopTestCase(TestCase):
-
-    @classmethod
-    def setUpTestData(cls):
-        # we don't want data created by data migrations
-        Product.objects.all().delete()
-
-    def test_smoketest(self):
-        self.assertTrue(True)
-
-    def test_notequal(self):
```

Comment:
Looks like not equal is actually still a thing in `db.py` could add this one back maybe?

Reply:
Yep, sounds good

Reply:
This has been added back